test/sor.test.ts

```diff
@@ -4,22 +4,25 @@ dotenv.config();
 import { SmartOrderRouter } from '../src/sor';
 import { sorGetSwapsWithPools } from '../src/static';
 import { SubgraphPoolProvider } from '../src/data/providers/subgraphPoolProvider';
-import { ChainId, ETH } from '../src/utils';
+import {
+    BALANCER_POOL_DATA_QUERIES_ADDRESSES,
+    ChainId,
+    ETH,
+} from '../src/utils';
 import { Token, TokenAmount } from '../src/entities';
 import { OnChainPoolDataEnricher } from '../src/data/enrichers/onChainPoolDataEnricher';
 import { SwapKind, SwapOptions } from '../src/types';
 import { BasePool } from '../src/entities/pools';
 
-const SOR_QUERIES = '0x1814a3b3e4362caf4eb54cd85b82d39bd7b34e41';
-
 describe('SmartOrderRouter', () => {
     describe('Mainnet', () => {
         const chainId = ChainId.MAINNET;
-        const rpcUrl = process.env['ETHEREUM_RPC_URL'] || '';
+        const rpcUrl =
+            process.env['ETHEREUM_RPC_URL'] || 'https://eth.llamarpc.com';
```

Comment:
I've added a fallback RPC as otherwise viem would need a chain passed along to figure out which public RPC to use. Could also change by passing the chain ID to the OnChainPoolDataEnricher.